nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -380,6 +389,10 @@ public boolean removeProperties(Namespace namespace, Set<String> properties) {
   }
 
   private boolean updateProperties(Namespace namespace, Consumer<Map<String, String>> action) {
+    if (namespace.isEmpty()) {
+      // As creating empty namespaces is not supported
+      throw new IllegalArgumentException("Empty namespaces not supported");
```

Comment:
this should be singular instead of plural. The same should probably be updated for the error msg that is thrown when an empty namespace is created

Reply:
updated